secrets/struct.go

```diff
@@ -0,0 +1,56 @@
+package secrets
+
+const (
+	DefaultSecretKey = "defaultsecretkey"
+	OverrideSecrets  = "override"
+	SecretKey        = "id"
+	SecretValue      = "secretvalue"
+	SecretType       = "store"
```

Comment:
Let's not introduce another keyword (store) in the APIs. Anyway every where we are calling it Type, so lets call it "type"